tests/PSRule.Rules.Azure.Tests/Cmdlet.Common.Tests.ps1

```diff
@@ -531,18 +531,227 @@ Describe 'Get-AzRuleTemplateLink' -Tag 'Cmdlet', 'Get-AzRuleTemplateLink' {
 
 #endregion Get-AzRuleTemplateLink
 
+#region Export-AzPolicyAssignmentData
+
+Describe 'Export-AzPolicyAssignmentData' -Tag 'Cmdlet', 'Export-AzPolicyAssignmentData' {
+    Context 'With Defaults' {
+        BeforeAll {
+            Mock -CommandName 'GetAzureContext' -ModuleName 'PSRule.Rules.Azure' -Verifiable -MockWith ${function:MockSingleSubscription};
+
+            Mock -CommandName 'Get-AzPolicyAssignment' -ModuleName 'PSRule.Rules.Azure' -Verifiable -MockWith {
+                return @(
+                    [PSCustomObject]@{
+                        Location           = 'eastus'
+                        Name               = '000000000000000000000000'
+                        ResourceId         = '/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/PolicyRG/providers/Microsoft.Authorization/policyAssign 
```

Comment:
@ArmaanMcleod Does this look correct?

Reply:
@BernieWhite Thanks for spotting that, missed that 😄.